<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HaveSnippetSettings</class>
 <widget class="QWidget" name="HaveSnippetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="apiUrlLabel">
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="apiUrlLineEdit">
     <property name="text">
      <string>http://your.havesnippet/</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Author's name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="shortDescriptionLabel">
     <property name="text">
      <string>Short description</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="shortDescriptionLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="langLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="langComboBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="fullDescriptionLabel">
     <property name="text">
      <string>Full description</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPlainTextEdit" name="fullDescriptionTextEdit">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="AutoCompleteLineEdit" name="tagsLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="accessibilityLabel">
     <property name="text">
      <string>Accessibility</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="accessibilityComboBox">
     <item>
      <property name="text">
       <string>Public</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unlisted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Logged users</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Private</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="expirationLabel">
     <property name="text">
      <string>Expiration</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="expirationComboBox">
     <item>
      <property name="text">
       <string>30 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keep forever</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="apiKeyLabel">
     <property name="text">
      <string>API key</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="apiKeyLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoCompleteLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/AutoCompleteLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
